README.md

```diff
@@ -1,11 +1,22 @@
-# OpenFisca Web-API
+# OpenFisca Web-API - DEPRECATED
 
-[![Build Status](https://travis-ci.org/openfisca/openfisca-web-api.svg?branch=master)](https://travis-ci.org/openfisca/openfisca-web-api)
+## This package is deprecated
 
-[More build statuses](https://www.openfisca.fr/build-status)
+**The official API is now available in [OpenFisca Core repository](https://github.com/openfisca/openfisca-core).**  
```

Comment:
**The OpenFisca Web API is now packaged in [OpenFisca-Core](https://github.com/openfisca/openfisca-core).**